javaScript.js

```diff
@@ -88,6 +88,34 @@ const projects = [
   },
 ];
 
+const form = document.getElementById('contactForm');
+const email = document.getElementById('email');
+const alert = document.getElementById('alertWin');
+const alertText = document.getElementById('alert-text');
+email.addEventListener = ('keyup', () => { alert.style.display = 'none'; });
+
+const getInfo = () => {
+  const formData = JSON.parse(localStorage.getItem('formData'));
+
+  if (formData) {
+    document.getElementById('full-name').value = formData.fullname;
+    document.getElementById('email').value = formData.email;
+    document.getElementById('message').value = formData.message;
+  }
+};
+
+form.addEventListener('keyup', () => {
+  const formData = {
+    fullname: document.getElementById('full-name').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
+  };
```

Comment:
- [ ] The contact form information is persistent to page reloads but when we load the information we cannot see any data so implement it as well.

Reply:
HELP MEEE PLS

Reply:
@roshan-bajgain can you review again pls